From a42674950f6f3ff87620d3261305753702946a06 Mon Sep 17 00:00:00 2001
From: zhang sanshan <b51434@freescale.com>
Date: Thu, 26 Nov 2015 10:13:07 +0800
Subject: [PATCH 571/602] MA-7255 BootAnimation starts delayed without
 disabled selinux of nand boot.(Auto Board)

The bootanimation starts with a long delay time for selinux blocking.
This patch add selinux policy for init.

Signed-off-by: zhang sanshan <b51434@freescale.com>
---
 imx6/sepolicy/init.te | 1 +
 imx7/sepolicy/init.te | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/imx6/sepolicy/init.te b/imx6/sepolicy/init.te
index c5a1c85..e76503d 100644
--- a/imx6/sepolicy/init.te
+++ b/imx6/sepolicy/init.te
@@ -8,3 +8,4 @@ allow init tmpfs:lnk_file  { create };
 allow init storage_file:dir  { mounton };
 allow init block_device:blk_file  { write };
 allow init sysfs_devices_system_cpu:dir  { write };
+allow init block_device:chr_file  { getattr ioctl };
diff --git a/imx7/sepolicy/init.te b/imx7/sepolicy/init.te
index 2ef28d9..e76503d 100644
--- a/imx7/sepolicy/init.te
+++ b/imx7/sepolicy/init.te
@@ -8,4 +8,4 @@ allow init tmpfs:lnk_file  { create };
 allow init storage_file:dir  { mounton };
 allow init block_device:blk_file  { write };
 allow init sysfs_devices_system_cpu:dir  { write };
-
+allow init block_device:chr_file  { getattr ioctl };
-- 
1.8.0

